omero/developers/Model/StructuredAnnotations.rst

```diff
@@ -100,21 +100,12 @@ annotation.
        TextAnnotation description = …;
        fileAnnotation.linkAnnotation(description);
 
-Immutability
```

Comment:
Also now further removal in https://github.com/openmicroscopy/ome-documentation/pull/1943.